server/channels/web/handlers.go

```diff
@@ -406,11 +406,34 @@ func (h Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			} else {
 				endpoint = h.HandlerName
 			}
-			c.App.Metrics().ObserveAPIEndpointDuration(endpoint, r.Method, statusCode, elapsed)
+
+			originDevice := "web"
+			if isMobile(r) {
+				originDevice = "mobile"
+			}
```

Comment:
I still wish we had a desktop differentiator as well. I understand the underlying client is pretty much the same as web but could still be helpful to understand where requests are coming from.

Reply:
Ah, yes, forgot about desktop at all :facepalm: Will do, thanks for the reminder!